modules/rostests/apitests/ntdll/LdrLoadDll.c

```diff
@@ -0,0 +1,107 @@
+/*
+ * PROJECT:     ReactOS API Tests
+ * LICENSE:     LGPL-2.0-or-later (https://spdx.org/licenses/LGPL-2.0-or-later)
+ * PURPOSE:     Test for LdrLoadDll
+ * COPYRIGHT:   Copyright 2017 Thomas Faber <thomas.faber@reactos.org>
+ *              Copyright 2024 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
+ */
+
+#define WIN32_NO_STATUS
+#include <wine/test.h>
+#include <pseh/pseh2.h>
+#include <ndk/rtlfuncs.h>
+#include <ndk/umfuncs.h>
+#include <ntstrsafe.h>
+
+/*
+
+NTSTATUS
+NTAPI
+LdrLoadDll(
+    IN PWSTR SearchPath OPTIONAL,
+    IN PULONG DllCharacteristics OPTIONAL,
+    IN PUNICODE_STRING DllName,
+    OUT PVOID *BaseAddress
+);
+
+*/
+
+#define StartSeh() \
+    ExceptionStatus = STATUS_SUCCESS; \
+    _SEH2_TRY
+
+#define EndSeh(ExpectedStatus) \
+    _SEH2_EXCEPT(EXCEPTION_EXECUTE_HANDLER) { \
+        ExceptionStatus = _SEH2_GetExceptionCode(); \
+    } _SEH2_END; \
+    ok(ExceptionStatus == ExpectedStatus, "Exception %lx, expected %lx\n", \
+       ExceptionStatus, ExpectedStatus)
```

Comment:
and these defines (and maybe the next one as well) can be removed.

Reply:
Removed.